carpool-map/css/l.geosearch.css

```diff
@@ -0,0 +1,75 @@
+.displayNone {
+	display: none;
+}
+
+.leaflet-control-geosearch {
+	position: relative;
+}
+
+.leaflet-control-geosearch a {
+	-webkit-border-radius: 4px;
```

Comment:
There's probably no need to use the vendor prefixes for these any more: http://caniuse.com/#feat=border-radius